src/tasks/node/NodeTaskHelper.ts

```diff
@@ -164,14 +164,15 @@ export class NodeTaskHelper implements TaskHelper {
     }
 
     public static async inferPackageName(packagePath: string): Promise<string> {
-        const packageJson = await fse.readFile(packagePath, 'utf8');
-        const packageContent = <NodePackage>JSON.parse(packageJson);
+        const packageContent = await fse.readJSON(packagePath);

```

Comment:
Use cast here to avoid unsafe-any lint warning below?

Reply:
D'oh!